libraries/chain/apply_context.cpp

```diff
@@ -224,4 +224,9 @@ const contracts::table_id_object& apply_context::find_or_create_table( name scop
    });
 }
 
+void apply_context::get_active_producers(account_name* producers, size_t datalen) {
+   const auto& gpo = controller.get_global_properties();
+   memcpy(producers, gpo.active_producers.producers.begin(), std::min(sizeof(account_name)*gpo.active_producers.producers.size(), datalen));
+}
```

Comment:
`gpo.active_producers.producers` is `fc::array<producer_key,config::producer_count> producers;` where `producer_key` is 
```
   struct producer_key {
      account_name      producer_name;
      public_key_type   block_signing_key;
```
So how can that be copied into an array of `account_name` ?